### ISSUE TITLE
Hostile NPC's Not Detecting Connected Player

### ISSUE THREAD

**Issue by Logickane** [SkyrimTogether/issues-game#186]
##Description

Players connected to host will experience hostile NPC's entering and exiting combat with them. As indicated on the compass at the top of the screen, red blip will disappear and reappear. While also saying things like "What's that?" and "Did you hear that?", things related to sneaking.

This also effects combat, as the enemy movements and attacks aren't working as intended. They may just stand there saying stuff, other times they may attack you.... kind of.

This was tested with and without mods. Mod list at the bottom.

##How to reproduce this issue:

1. Start the game.
2. Connect to the server.
3. Connect to another player and have them host the cell.
4. Run up to some hostile enemies, have the host stay a little ways away so they aren't detected.

##How many times can you reproduce it?

Currently, every time, anywhere. Same inconsistencies. 

## Expected result

Enemy will detect you, and engage, with consistent attacks as you would expect from Vanilla Skyrim. (Wonky desync'd animations aside.)

## Your environment

The Elder Scrolls V: Skyrim Special Edition
SKSE
SkyUI - Mod
Alternate Life - Mod

**Comment by HFinner** [comment]
Also having this issue! Only the host is attacked by enemies and you are completely ignored. The biggest issue for sure!

**Comment by Logickane** [comment]
Yeah, I can deal with a phew extra NPC's or some not actually being there, but this is breaking it for me. What's the point if I can stand in their face and they stand there saying "Who's there?". I can actually even stealth and if I time it right when the red blip disappears on the compass I can get a damage bonus for stealth lol.

**Comment by Logickane** [comment]
Following up with this issue and letting you know that it's still in the game. Just tested with current version.
